src/Codec/FFmpeg/Resampler.hs

```diff
@@ -0,0 +1,149 @@
+module Codec.FFmpeg.Resampler where
+
+import           Codec.FFmpeg.Common
+import           Codec.FFmpeg.Enums
+import           Codec.FFmpeg.Types
+import           Control.Concurrent.STM.TChan
+import           Control.Monad                (void, when)
+import           Control.Monad.STM
```

Comment:
This is more out of curiosity than anything else: what is the thinking behind using `stm` for the channel? It seems like a fine choice since it is relatively well-known and has a limited list of dependencies, but I'm curious if there's anything else to argue for it over other the `Chan` in base given that the use here doesn't obviously look particularly risky.